calcom/cal.com
# fix: correct language is used when booking with a modal + location icon visible in booker when using light mode

## What does this PR do?

This PR fixes the following:
- The "location" icon is now visible in light move when using the "in person (Organizer Address)" and "Show On Booking Screen" option
- The language in which the date and length of an event is shown in modals now corresponds to the language of the creator of the calendar (before this is was always in english) and also shows the unit in which the event length is indicated (before this, only a number was shown)

- If the text is too wide for the modal (which before this never happened because it was always in English), they will adapt into a column

- Refer to the screenshots to get a better idea of the impact of this PR.

Before screenshots:
<img width="423" alt="Screenshot 2024-04-01 at 13 13 38" src="https://github.com/calcom/cal.com/assets/41197000/287562f4-7e57-4422-8249-c2075cc3edc6">
<img width="558" alt="Screenshot 2024-04-01 at 13 14 24" src="https://github.com/calcom/cal.com/assets/41197000/4a4ade04-f19d-4dd1-97c4-3eba07f43540">

After screenshots:
<img width="421" alt="Screenshot 2024-04-01 at 13 14 41" src="https://github.com/calcom/cal.com/assets/41197000/e45146a1-3946-4ebe-a7db-d18d444f7d1f">
<img width="562" alt="Screenshot 2024-04-01 at 13 14 48" src="https://github.com/calcom/cal.com/assets/41197000/8aff6934-78b9-49d6-b053-c382c3bc13be">
<img width="352" alt="Screenshot 2024-04-01 at 13 15 36" src="https://github.com/calcom/cal.com/assets/41197000/57a027d7-c4ba-4bfa-b453-2ee6687bed84">

PS: please note that the word "minute" is the same in English and French so it kind of looks like it isn't using the correct language in the screenshots but it is :). Probably better to test with another language than French

## Requirement/Documentation

N/A

## Type of change



- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

Book an appointment with the column view in any other language than English while using light mode. Be sure to include an "in person (Organizer Address" address so you can confirm that the "location" icon is visible.
